apps/perf-test/src/scenarioRenderTypes.js

```diff
@@ -8,11 +8,12 @@
  * memoization logic help avoid certain code paths.
  */
 
-const AllRenderTypes = ['mount', 'virtual-rerender'];
+const AllRenderTypes = ['mount', 'virtual-rerender', 'virtual-rerender-with-unmount'];
```

Comment:
out of scope but it seems to me like the extensible way of doing this is to add some kind of configuration to the render type. in this case a `withUnmount` option for `virtual-rerender`